pages/osx/dd.md

```diff
@@ -1,19 +0,0 @@
-# dd
```

Comment:
Also the units for bs=2m is capitalized on one and not the other.
The gnu and BSD implementations of dd are sufficiently different imo
On Jul 25, 2016 06:14, "Waldir Pimenta" <notifications@github.com> wrote:

In pages/osx/dd.md
<https://github.com/tldr-pages/tldr/pull/977#discussion_r72039508>:

> @@ -1,19 +0,0 @@
> -# dd

That difference is justified: I've tested status=progress on OSX and it
results in "dd: unknown operand status". We should either remove
status=progress from the common page, or keep the separate osx page.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tldr-pages/tldr/pull/977/files/071da22125918034aba0c10592e2dc751f8c87f2#r72039508>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAHkXrfbjUQ2Dd-yNuTE64xH2Y3NrPyoks5qZIyNgaJpZM4JTyao>
.